docs/examples/beneficial_ownership/beneficial_owners_extension.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Let's use a consistent order for package metadata and release fields, so that files are easier for users to visually compare. The other file has a good order.